ci/before_install.sh

```diff
@@ -23,12 +23,17 @@ echo 'python info:'
   2>&1 pyenv versions || true
 ) | sed 's/^/  /'
 
+if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
+    echo "Install Python 2."
+    brew install python@2
+    echo "Upgrade Python 3."
```

Comment:
Upgrading Python  3 could also be moved to the next `if` block below, if you prefer.